lib/models/event.js

```diff
@@ -142,5 +145,13 @@ module.exports.MatrixEvent.prototype = {
      */
     isState: function() {
         return this.event.state_key !== undefined;
+    },
+
+    getPushActions: function(client) {
+        if (this.pushActions === undefined) {
+            var pushProcessor = new PushProcessor(client);
+            this.pushActions = pushProcessor.actionsForEvent(this.event);
+        }
+        return this.pushActions;
```

Comment:
Nitpick: I assume we do not want to expose `this.pushActions`, so it would be nice to prefix with an `_` e.g. `this._pushActions` to make this clear.